docs/examples/hydration-phenol-explicit.rst

```diff
@@ -0,0 +1,127 @@
+.. _hydration_phenol_explicit:
+
+Hydration Free Energy of Phenol in Explicit TIP3P Water
```

Comment:
Is this example really separate? It's not just a subset of the FreeSolv dataset?